src/components/pages/ParentNewsFeed/index.js

```diff
@@ -4,21 +4,20 @@ import '../../../styles/index.less';
 import NewsContainer from './NewsContainer';
 import ParentSidebar from '../ParentHome/ParentSidebar';
 import Banner from '../../common/Banner';
-import TitleParentNewsFeed from './TitleParentNewsFeed';
+// import TitleParentNewsFeed from './TitleParentNewsFeed';
```

Comment:
Is this something you will work on in the nearest future?

Reply:
this was removed in another PR actually- good catch!